src/app/course/practice-problem/practice-problem.component.html

```diff
@@ -11,55 +11,28 @@
                     size="s"
                     tuiButton
                 >
-                    Back to concept map
+                    Back to Concept Map
+
                 </a>
             </div>
+            
             <div
-                class="tui-space_bottom-2"
-            >
-                <span
-                    tuiHintPointer
-                    tuiHint="Coming Soon..."
-                    tuiHintDirection="top-middle"
-                >
-                    <a
-                        [routerLink]="['../../../goal']"
-                        class="tui-space_vertical-0 w-full"
-                        icon="tuiIconArrowLeft"
-                        size="s"
-                        tuiButton
-                        disabled="true"
-                    >
-                        Back to goals
-                    </a>
-                </span>
-            </div>
-            <div
-                *ngIf="parentCategory"
-                class="practice-sidebar_content-breadcrumbs tui-space_bottom-2"
+                class="tui-space_bottom-2 tui-col_xs-12 tui-col_md-12 tui-col_lg-12"
             >
                 <a
-                    [class.tui-skeleton]="!category"
-                    [class.tui-skeleton_text]="!category"
-                    [routerLink]="['../', parentCategory?.pk]"
-                    class="tui-space_vertical-0"
-                    tuiLink
+                    [routerLink]="['../../../']"
```

Comment:
Why does the URL of `back to goals` changed to this?

Reply:
It was a placeholder link because we said just to link it anywhere for now. I'm not sure how to link it to the goals page though, I'll ask.